docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/EntityFrameworkEnumerable.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class HousingData
+    {
+        // Entity Framework requires the object to have a primary key
+        public int HousingDataId { get; set; }
+        public float MedianHomeValue { get; set; }
+        public float CrimesPerCapita { get; set; }
+        public float PercentResidental { get; set; }
+        public float PercentNonRetail { get; set; }
+        public float CharlesRiver { get; set; }
+        public float NitricOxides { get; set; }
+        public float RoomsPerDwelling { get; set; }
+        public float PercentPre40s { get; set; }
+        public float EmploymentDistance { get; set; }
+        public float HighwayDistance { get; set; }
+        public float TaxRate { get; set; }
+        public float TeacherRatio { get; set; }
+    }
+
+    public partial class MLNetExampleContext : DbContext
```

Comment:
>partial [](start = 11, length = 7)

Is this a partial class, or is it all in this file?

Reply:
From looking at the code, it seems this became partial because it was generated by reverse engineering the database. Partial definitively can be removed.